lib/ansible/plugins/callback/__init__.py

```diff
@@ -98,6 +98,9 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
         # load from config
         self._plugin_options = C.config.get_plugin_options(get_plugin_class(self), self._load_name, keys=task_keys, variables=var_options, direct=direct)
 
+    def _run_is_verbose(self, result):
+        return ((self._display.verbosity > 0 or '_ansible_verbose_always' in result._result) and '_ansible_verbose_override' not in result._result)
```

Comment:
I know this is preexisting issue, but the condition should be checking the boolean value, not just the existence

Reply:
get it